planning march 2021/userStoriesSummary.md

```diff
@@ -0,0 +1,145 @@
+## Features
+
+### Org employee
+
+- [ ] Create user without an organisation - 1d
+- [ ] Join organisation - 1
+- [ ] Edit user details - 2
+- [ ] Should only see/edit application for their company and user - 1
+- [ ] Do updates on applications after "changes required" - 1
+- [ ] Get notified by email when "changes required" / "submitted" / "outcome changes" - 1
+- [ ] See application with different outcomes - 1
+- [ ] Request admin permission in company - 3
+- [ ] Apply for drug registration - 1
+- [ ] And import permit - 2
+- [ ] Receive evidence of drug registration and import permit - 2
+- [ ] Withdrawn application (less priority) - 3
+- [ ] Delete a draft application - 2
+- [ ] Select different organisation to apply for Drug registration and import permit - 2d
+- [ ] View applications by status "submitted", by dates and by outcome - 1
+- [ ] View applications deadline which are waiting for "changes required" - 2
+- [ ] Duplicate/Export application - 3
+
+### Org owner
+
+- [ ] Create a company - 1
+- [ ] Display company logo from application - 2
+- [ ] Apply to make changes to organisation - 3
+- [ ] Approve user joining organisation - 1
+- [ ] View users permission on organisation - 2
+- [ ] Edit (add/delete) user permission on organisation - 2
+- [ ] Remove user from organisation - 3
+- [ ] View organisation status - available/disabled, - renew date - 3
+- [ ] List of application results (i.e. import permits expiry dates) - 1
+- [ ] Apply for more then one company - 2
+- [ ] Notifications for upcoming renewals (drug rego, import permits, etc..) - 3
+
+### Reviewer
```

Comment:
we should add something about being able to view history for a particular question